### PR TITLE
Feature/rounds

### PR DESCRIPTION
**Pull request by hughwin**
Added a round counter in the game class to keep track of what round the current game is in.

### CODE REVIEW

File: src/main/java/com/toptrumps/cli/CommandLineUI.java [hunk 1]
```diff
@@ -141,4 +143,11 @@ public void showActivePlayer(){
         System.out.println("The active player is: " + activePlayerName);
     }
 
+    public void showRound(){
+        String round = "Round " + game.getRoundCounter();
+        System.out.println(round);
```
Comment (hughwin) [inline src/main/java/com/toptrumps/cli/CommandLineUI.java:149]: I know someone will comment that it does the same thing twice, but it's how it is in the example CLI output. We can change it later.

Comment (BelfDev) [reply src/main/java/com/toptrumps/cli/CommandLineUI.java:149]: OK

File: src/main/java/com/toptrumps/cli/CommandLineUI.java [hunk 2]
```diff
@@ -40,8 +48,17 @@ public void startGame() {
         logger.printToLog("New line \n new line too.");
         input = scanner.nextLine();
         if (input.equalsIgnoreCase("f")) {
+            user = new Player("Human");
```
Comment (BelfDev) [inline src/main/java/com/toptrumps/cli/CommandLineUI.java]: I'd reconsider this design decision. We already know that there's only one Human Player and this logic applies to both Apps. The Game object was supposed to encapsulate all the game logic, including "Player".

File: src/main/java/com/toptrumps/core/Game.java [hunk 3]
```diff
@@ -65,4 +69,22 @@ public void nextPlayer(){
...
+     */
+    public void setSelectedAttribute(Attribute selectedAttribute){
+        this.selectedAttribute = selectedAttribute;
+    }
+
+    /**
+     * Method to increase the roundCounter by 1
+     */
+
+    public void incrementRoundCounter(){roundCounter++;}
```
Comment (BelfDev) [inline src/main/java/com/toptrumps/core/Game.java:85]: Is this **method** really necessary?

**Review by BelfDev — APPROVED** [review]
# Summary

I have a few additional considerations but I'll revise them in another branch. I want to get this to the development! :)

**Review by rjs68 — APPROVED** [review]

**Pull request merged**